docarray/documents/helper.py

```diff
@@ -118,3 +118,44 @@ class MyAudio(TypedDict):
     doc = create_model_from_typeddict(typeddict_cls, **kwargs)
 
     return doc
+
+
+def create_from_named_tuple(
+    named_tuple_cls: Type['NamedTuple'],
+    **kwargs: Any,
+):
```

Comment:
```suggestion
def create_from_named_tuple(
    named_tuple_cls: Type['NamedTuple'],
    **kwargs: Any,
): -> Type['BaseDocument']
```

Reply:
I had one doubt though, should it be `Type['BaseDocument']` or `Type['T_doc']`?